CMakeLists.txt

```diff
@@ -132,8 +132,41 @@ endif()
 
 set(BOOST_DIRECTLY_REQUIRED_LIBRARIES filesystem system regex program_options)
 
+find_program(PKG_CONFIG_EXECUTABLE NAMES pkgconf.exe pkg-config.exe)
 find_package(PkgConfig)
 
+# Note: Saving CMAKE_MODULE_PATH as CGAL will overwrite it.
+# Reconsider this after CGAL 5.4: https://github.com/CGAL/cgal/pull/6029
+if ("${CMAKE_MODULE_PATH}" STREQUAL "")
```

Comment:
This whole section is doing some odd things.  `CMAKE_MODULE_PATH` Is actually a list of paths to search over, not a single path, so I don't think there should be any check for empty string.  
The previous behavior was actually appending a path to the existing value, instead of completely replacing it.  

If you look at the automated builds (click the red X next to your commits), you can see that MXE builds have been failing to find libzip since your changes.  This is likely because the MXE toolchain sets its own CMAKE_MODULE_PATH, and so our additional path is not appended here.

If a platform/toolchain, such as VCPKG or MXE provides `Find[package].cmake` files etc, its best to use those before our own.  (Ideally we wouldn't need to maintain our own *at all*, if every platform had all the needed find modules).